modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/profile/components/InterestsCard.jsx

```diff
@@ -31,7 +30,8 @@ export const InterestsList = ({channelId, groupId, id, interests}) => (
 			<ListGroup.Item className='interest' key={name}>
 				<ListGroup.ItemTitle className='text-truncate'>
 					{name ? (
-						<Link
+						<ClayLink
+							decoration='none'
 							to={toRoute(
```

Comment:
Hey @fmarcio, ClayLink doesn't have `to` attr. Please change it to use `href` instead.

Reply:
@interaminense done!